src/topology/continuous_function/continuous_section.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright © 2021 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nicolò Cavalleri
+-/
+
+import topology.continuous_function.basic
+import data.bundle
+
+/-!
+# Continuous sections
+
+In this file we define bundled continuous sections of a map.
+
+-/
+
+section
+
+set_option old_structure_cmd true
+
+variables {α: Type*} {β: Type*} {f : α → β}
+variables [topological_space α] [topological_space β]
+
+/-- A continuous section of a map `f` is a continuous map that is a right inverse of `f`.
+Note that the `nolint has_inhabited_instance` here has a true matematical meaning: the universal
```

Comment:
the `nolint` does not mean this: the linter wants *one* example where the structure is inhabited, possibly with more structure.

Reply:
Do you have any suggestions for what inhabited instance to put? Otherwise I will live the `nolint` but remove the comment

Reply:
Presumably you can provide an inhabited instance for `continuous_section (id : α → α)`?

Reply:
Well I guess the reason I wrote that comment is that there is written something similar for homeomorphisms, perhaps that should be changed so not to confuse other people by me.

To be fair I would actually take out the comment but leave the nolint at this point... I don't see any advantage in having inhabited instances for very specific cases.